### DISCUSSION TITLE
Auto Change keys

### DISCUSSION THREAD

**Discussion by robertbigk** [ericaltendorf/plotman#886] [Q&A]
![msedge_PcUVO2j6ii](https://user-images.githubusercontent.com/34956477/126561175-f574e28d-b22c-46bd-ab1b-7409613153fc.png)

Hi I plot for my friend he got all computers. How to disable auto change keys every time when I shut down machinaris .

**Comment by altendky** [comment]
machinaris uses plotman, but I think you need to check with them about this, sorry.  Take a look over at https://github.com/guydavis/machinaris.

**Reply by robertbigk** [reply]
ahh i already ask machinaris and get answer You need to check Plotman

**Reply by guydavis** [reply]
oops!  There appears to be a communication problem here.  At no time did I indicate the issue was in Plotman.  @altendky   Apologies for any confusion.

**Reply by altendky** [reply]
Heh, we'll get there.  `:]`

**Reply by robertbigk** [reply]
Than for now is no any way to resolve this is no matter or how to ?

**Reply by altendky** [reply]
I don't have any context here to know where the conversation started or how to follow up and make sure we're all on the same page (the one with the solution, that is).  Any leads for me?
